packages/@react-spectrum/combobox/test/ComboBox.test.js

```diff
@@ -121,6 +123,73 @@ describe('ComboBox', function () {
     expect(label).toBeVisible();
   });
 
+  it('can be disabled', function () {
+    let {getByRole} = renderComboBox({isDisabled: true});
+
+    let combobox = getByRole('combobox');
+    act(() => {
+      combobox.focus();
+      userEvent.type(combobox, 'One');
+      jest.runAllTimers();
+    });
+
+    expect(() => getByRole('listbox')).toThrow();
+    expect(onOpenChange).not.toHaveBeenCalled();
+    expect(onFocus).not.toHaveBeenCalled();
+
+    act(() => {
+      fireEvent.keyDown(combobox, {key: 'ArrowDown', code: 40, charCode: 40});
```

Comment:
please include a keyUp as well, we've been bitten by that one before where things aren't cleaned up in tests correctly because the chain of events that would ordinarily happen don't complete